source/MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/OgsFxShaderGenerator.cpp

```diff
@@ -1,41 +1,91 @@
 #include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/OgsFxShaderGenerator.h>
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/PositionOgsFx.h>
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/NormalOgsFx.h>
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/TangentOgsFx.h>
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/BitangentOgsFx.h>
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/TexCoordOgsFx.h>
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/AdskSurfaceOgsFx.h>
-#include <MaterialXShaderGen/ShaderGenerators/Glsl/OgsFx/SurfaceOgsFx.h>
-#include <MaterialXShaderGen/HwShader.h>
 #include <MaterialXShaderGen/Syntax.h>
 
 #include <sstream>
 
 namespace MaterialX
 {
 
+namespace
+{
+    // Semantics used by OgsFx
+    static const std::unordered_map<string,string> OGSFX_DEFAULT_SEMANTICS_MAP =
+    {
+        { "i_position", "POSITION"},
+        { "i_normal", "NORMAL" },
+        { "i_tangent", "TANGENT" },
+        { "i_bitangent", "BITANGENT" },
+        { "i_texcoord0", "TEXCOORD0" },
+        { "i_texcoord1", "TEXCOORD1" },
+        { "i_texcoord2", "TEXCOORD2" },
+        { "i_texcoord3", "TEXCOORD3" },
+        { "i_texcoord4", "TEXCOORD4" },
+        { "i_texcoord5", "TEXCOORD5" },
+        { "i_texcoord6", "TEXCOORD6" },
+        { "i_texcoord7", "TEXCOORD7" },
+        { "i_texcoord8", "TEXCOORD8" },
+        { "i_texcoord9", "TEXCOORD9" },
+        { "u_modelMatrix", "World" },
+        { "u_viewProjectionMatrix", "ViewProjection" },
+        { "u_normalMatrix", "WorldInverseTranspose" },
```

Comment:
"normalMatrix" was a convention used in a couple of shaders I saw, I guess since this is normally only used for normal vector transformation. But I'm fine with naming it "worldInverseTranspose" instead, since that's technically what it is. In that case maybe we should change "modelMatrix" to "worldMatrix" as well to be consistent.